### DISCUSSION TITLE
Grafana to OpenObserve Dashboard Converter

### DISCUSSION THREAD

**Discussion by kubserv** [openobserve/openobserve#2885] [Show and tell]
Hey,
I have created a very crude (but working) dashboard converter, because I migrated from the Prometheus/Loki/Grafana stack to OpenObserve. Since I mostly used public dashboards, I wrote a converter to help me migrate them.

[https://github.com/kubserv/converter](https://github.com/kubserv/converter)

It's in a very early stage and therefore not feature-complete yet. You will need to do some editing, once you imported the generated dashboard to OpenObserve, but it saved me a lot of time. This way I can search for Grafana Dashboards and then use them in OpenObserve.

**Comment by prabhatsharma** [comment]
This is awesome.
